### PR TITLE
Translated workshop Prerequisites section in Japanese

### PR DESCRIPTION
**Pull request by cmick-aws**
@tkimurz Please review Japanese translation.

### CODE REVIEW

File: content/prerequisites/account/index.ja.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## 検証のために AWS アカウントを作成
+
+アプリケーションをデプロイするために、AWS アカウントへのアクセスが必要です。すでにアカウントをお持ちで、さらに管理者権限を持っているユーザが利用する環境に設定されている場合は、[次のステップ](../awscli)に移動しても大丈夫です。
+
+:::alert{type="warning"}
+既存の個人アカウントまたは会社のアカウントを利用する場合は、そのアカウントにリソースを作成する場合の影響とポリシーをご確認ください。
+:::
+
+AWS アカウントを持ちでない場合は、[こちらで無料でAWS アカウントを作成](https://portal.aws.amazon.com/billing/signup)することができます。
```
Comment (uchiroki) [inline content/prerequisites/account/index.ja.md:14]: Before: することができます
After: できます

File: content/prerequisites/awscli/index.ja.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+aws configure
+:::
+
+[前のステップ](/prerequisites/account)で作成した **アクセスキー ID** と **シークレットアクセスキー** を入力し、デフォルトのAWS リージョン (例えば `ap-northeast-1`) を入力します。できれば、まだリソースが作成されていないリージョンが望ましいです。
+
+:::code{language=shell showLineNumbers=false showCopyAction=false}
+AWS Access Key ID [None]: <アクセスキー ID を入力>
+AWS Secret Access Key [None]: <シークレットアクセスキーを入力>
+Default region name [None]: <リージョンを入力 (e.g. "us-east-1", "ap-northeast-1")>
+Default output format [None]: <空欄で結構>
```
Comment (uchiroki) [inline content/prerequisites/awscli/index.ja.md:24]: Before: 空欄で結構
After: 空欄のまま

File: content/prerequisites/editor/index.ja.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: "コードエディタのインストール"
+weight: 300
+---
+
+[YAML](https://yaml.org/) の編集をサポートするお好みのコードエディタや IDE を利用できますが、このワークショップでは macOS、Linux、Windows でうまく動作する [Visual Studio Code](https://code.visualstudio.com/) の利用を前提とします。
+
+VS Code をインストールするためには、オペレーティングシステムのパッケージマネージャ（例： macOS の `brew cask install visual-studio-code`）を使うか、[VS code のサイトの指示](https://code.visualstudio.com/) に従ってインストールしてください。
```
Comment (uchiroki) [inline content/prerequisites/editor/index.ja.md:8]: Before: VS code
After: VS Code
本家から変えたほうが良い箇所の一つです

**Review by uchiroki — COMMENTED** [review]
翻訳ありがとうございます！ 概ね問題ございません。細かい部分の修正をお願いします。

File: content/prerequisites/account/index.ja.md (hunk 1 above)
Comment (cmick-aws) [reply content/prerequisites/account/index.ja.md:14]: Fixed.

File: content/prerequisites/awscli/index.ja.md (hunk 2 above)
Comment (cmick-aws) [reply content/prerequisites/awscli/index.ja.md:24]: Fixed

File: content/prerequisites/editor/index.ja.md (hunk 3 above)
Comment (cmick-aws) [reply content/prerequisites/editor/index.ja.md:8]: Fixed.

I will also ask to a fix on EN version.

**Review by uchiroki — DISMISSED** [review]
I also checked the fixed version . Finished reviewing.

**Comment by cmick-aws** [comment]
@rezabekf This PR has been reviewed in Japan side. Please squash merge.

**Comment by cmick-aws** [comment]
@uchiroki , I also localized URLs for this part of the workshop, as decided. Please review.

**Review by uchiroki — APPROVED** [review]
I reviewed the latest version. There is no problem at all.

**Review by rezabekf — APPROVED** [review]
LGMT 👍

**Pull request merged**